crates/config/src/zksync.rs

```diff
@@ -146,8 +147,22 @@ impl ZkSyncConfig {
             suppressed_errors: self.suppressed_errors.clone(),
         };
 
+        let zksolc_path = if let Some(path) = config_ensure_zksolc(self.zksolc.as_ref(), offline)? {
```

Comment:
is this the only place where this procedure (find & install if needed) is present now?

Reply:
Yup! If you ask precisely how many times we call `config_ensure_zksolc `, this is the only time we do it now. 
However, if you ask each functionality (find & install) individually, we call in many other places. Use `ZkSolc::find_installed_version` and `ZkSolc::blocking_install` as kw references to see where specifically.